conda/envs/legacy/pixi.toml

```diff
@@ -0,0 +1,64 @@
+[project]
+name = "gazebo"
+version = "0.1.0"
+description = "Pixi configuration to replicate existing Windows build environment in Gazebo"
+authors = ["Jose Luis Rivero <jrivero@openrobotics.com>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "linux-aarch64", "win-64"]
+
+[dependencies]
+assimp = "*"
+boost-cpp = "1.78.0.*"
+bullet-cpp = "*"
+cppzmq = "*"
+cuda = "*"
+curl = "*"
+dartsim = "6.13.2.*"
+eigen = "*"
+fcl = "0.7.0.*"
+ffmpeg = "*"
+freeimage = "*"
+gdal = "*"
+gflags = "*"
+glib = "*"
+gts = "*"
+jsoncpp = "*"
+libccd-double = "*"
+libprotobuf = "*"
+libsqlite = "*"
+libzip = "*"
+ogre = "1.10.12.1.*"
+ogre-next = "2.3.3.*"
```

Comment:
The current version in vcpkg is 2.2.x. Using 2.3.3 here will break Fortress builds. Should we not have `envs/fortress`, `envs/harmonic`,  `envs/ionic`, and `envs/jetty`?

Reply:
This first step tries to reproduce the legacy environment created by vcpkg and afterwards to create new environments for each (new?) distribution. The reason is not to alter too much the current status of the existing Windows CI builds with respect about how they are on vcpkg. I changed the version in [211a78a](https://github.com/gazebo-tooling/release-tools/pull/1192/commits/211a78aaec76c9e1aa3bd9f4b094abd539443d63)

Reply:
Okay, that makes sense. For the follow-up work, I would really like gz-rendering CI for Harmonic and Ionic to run with Ogre 2.3.3 and resolve https://github.com/gazebosim/gz-rendering/issues/745 so it would be great if we created new environments for existing distributions as well as new ones.

Reply:
Understood. So custom environments start with Harmonic and Ionic, maybe the same than legacy but using ogre-2.3.3 ?

Reply:
Yes, I think that's right. The only other version that might be different is CMake, but I would try with the latest first and see if we have any problems.